src/main/java/domain/ApplicationConfig.java

```diff
@@ -0,0 +1,45 @@
+package domain;
+
+public class ApplicationConfig {
+    private Boolean debugLogging;
+    private String timezone;
+    private Integer repeatIntervalSeconds;
+
+    public ApplicationConfig(String[] args) throws Exception {
+        for (int i = 0; i < args.length; i++) {
+            if (args[i].equals("--timezone")) {
```

Comment:
hardcoding parameter values is a NO NO what were you thinking ?? - try using constants ?